integrations/discord/events.go

```diff
@@ -7,19 +7,51 @@ import (
 	"go.uber.org/zap"
 
 	"go.autokitteh.dev/autokitteh/integrations/discord/internal/vars"
+	"go.autokitteh.dev/autokitteh/sdk/sdktypes"
 )
 
 func (h *handler) handleEvent(event any, eventType string) {
-	akEvent, err := h.transformEvent(event, eventType)
-	if err != nil {
+	var authorID string
+
+	// TODO(ENG-1546) - Add support for more event types
+	switch e := event.(type) {
+	case *discordgo.MessageCreate:
+		authorID = e.Author.ID
+	case *discordgo.MessageUpdate:
+		authorID = e.Author.ID
+	case *discordgo.MessageDelete:
+		authorID = e.Author.ID
```

Comment:
```suggestion
	case *discordgo.MessageCreate, *discordgo.MessageUpdate, *discordgo.MessageDelete:
		authorID = e.Author.ID
```

Reply:
<img width="852" alt="image" src="https://github.com/user-attachments/assets/fe16d60f-0284-4041-bd32-1507b825bd1a">
I tried that and go complains

Reply:
Oops, my bad. I forgot it's a type switch.